### PR TITLE
Added File Manager Helpers

### PR DESCRIPTION
**Pull request by jumaallan**
Story: https://app.shortcut.com/smileid/story/xxx

## Summary

A few sentences/bullet points about the changes

## Known Issues

Any shortcomings in your work. This may include corner cases not correctly handled or issues related
to but not within the scope of your PR. Design compromises should be discussed here if they were not
already discussed above.

## Test Instructions

Concise test instructions on how to verify that your feature works as intended. This should include
changes to the development environment (if applicable) and all commands needed to run your work.

## Screenshot

If applicable (e.g. UI changes), add screenshots to help explain your work.

### CODE REVIEW

File: Sources/SmileID/Classes/Helpers/LocalStorage.swift [hunk 1]
```diff
@@ -199,6 +199,28 @@ public class LocalStorage {
...
 
+    static func getUnsubmittedJobs() -> [String] {
+        var fileNames: [String] = []
+        do {
+            let files = try fileManager.contentsOfDirectory(atPath: unsubmittedDirectory.relativePath)
+            fileNames.append(contentsOf: files)
+            return fileNames
+        } catch {
+           return fileNames
+        }
```
Comment (vanshg) [inline Sources/SmileID/Classes/Helpers/LocalStorage.swift:210]: No need for the extra allocation. Let's also add a log for the error case

```suggestion
        do {
            return try fileManager.contentsOfDirectory(atPath: unsubmittedDirectory.relativePath)
        } catch {
           print("Error fetching unsubmitted jobs: \(error.localizedDescription)")
           return []
        }
```

File: Sources/SmileID/Classes/Helpers/LocalStorage.swift [hunk 2]
```diff
@@ -199,6 +199,28 @@ public class LocalStorage {
...
+    }
+
+    static func getSubmittedJobs() -> [String] {
+        var fileNames: [String] = []
```
Comment (vanshg) [inline Sources/SmileID/Classes/Helpers/LocalStorage.swift:214]: ditto

File: Sources/SmileID/Classes/SmileID.swift [hunk 3]
```diff
@@ -64,6 +65,37 @@ public class SmileID {
...
+    }
+
+    /// nukes all offline stored files
+    public class func cleanup() throws {
+        try LocalStorage.deleteAll()
+    }
+
+    /// deletes the passed job id, whether in unsubmitted or submitted state
+    public class func cleanup(jobId: String) throws {
+        try cleanup(jobIds: [jobId])
```
Comment (vanshg) [inline Sources/SmileID/Classes/SmileID.swift:91]: It is unfortunate that Swift does not support splatting -- that would have allowed us to have 2 methods here instead of 3.

I noticed we could easily provide more functionality here by making this a variadic function.

```suggestion
    public class func cleanup(jobIds: String...) throws {
        try cleanup(jobIds: jobIds)
```

This would allow any of the following to be valid:
```swift
try cleanup(jobIds: "job_1")
try cleanup(jobIds: "job_1", "job_2")
try cleanup(jobIds: ["job_1", "job_2"])
```

Comment (vanshg) [reply Sources/SmileID/Classes/SmileID.swift:91]: We could also do something like this:
```swift
    /// deletes the job ids list, whether in unsubmitted or submitted state
    public class func cleanup(jobIds: [String]? = nil) throws {
        if let jobIds {
            try LocalStorage.delete(at: jobIds)
        } else {
            try LocalStorage.deleteAll()
        }
    }

    public class func cleanup(jobIds: String...) throws {
        try cleanup(jobIds: jobIds)
    }
```

File: Sources/SmileID/Classes/Helpers/LocalStorage.swift (hunk 1 above)
Comment (jumaallan) [reply Sources/SmileID/Classes/Helpers/LocalStorage.swift:210]: fixed

File: Sources/SmileID/Classes/Helpers/LocalStorage.swift (hunk 2 above)
Comment (jumaallan) [reply Sources/SmileID/Classes/Helpers/LocalStorage.swift:214]: fixed

File: Sources/SmileID/Classes/SmileID.swift (hunk 3 above)
Comment (jumaallan) [reply Sources/SmileID/Classes/SmileID.swift:91]: cleaned up

**Review by vanshg — APPROVED** [review]
